citadel/ros_integration.md

```diff
@@ -74,9 +78,11 @@ setup.bash (not needed for binary packages):
 
 ```bash
 # Shell D:
-. ~/bridge_ws/install/setup.bash
+. ~/workspace/install/setup.bash
 ```
 
+**Note**: The `workspace` directory is the name of the workspace you created (e.g., in [Ubuntu Source Install](install_ubuntu_src)), which can be named anything.
```

Comment:
a bit late but we should update this link to `[install section](https://github.com/osrf/ros_ign/tree/melodic/ros_ign_bridge#building-the-bridge-from-source)` since the instructions are referring to the ign_ros bridge workspace and not the citadel workspace.

Reply:
Should I change the `workspace` directory back to `bridge_ws` then? For me, the small confusion arose after I followed the install from source section then following this (ros integration) tutorial afterwards. When I cloned ros_ign I put it in the `workspace` directory.

Reply:
I see. I think that's something we can clarify. We can mention that this is the workspace where `ros_ign` is in, which can, but not necessarily, be the same workspace as where the citadel is built in.

Reply:
> a bit late

Sorry I pulled the trigger too fast

> we should update this link to [install section]

There haven't been any install instructions within the `ros_ign_bridge` directory since https://github.com/ignitionrobotics/ros_ign/pull/72/, see also this issue https://github.com/ignitionrobotics/ros_ign/issues/90.

How about to avoid confusion, we don't provide a source command, and instead just explain: "Make sure you've sourced ROS and the ros-ign packages.". Someone running debs may not even have a workspace to source.

Reply:
I think that works, there are already areas in the tutorials that do the same (i.e., states to source the package but doesn't provide the command). Should I go ahead and make this change?

Reply:
That sounds good to me, but let's wait for Ian's ok.

Reply:
yep sounds good to me too

Reply:
Should I remove the commands to source the ROS workspace that are listed under `# Shell A`, `# Shell D`, `# Shell E` as wwell?

Reply:
I think sourcing the ros setup files should be fine, and we can leave them there. Users need to source the ROS setup file even if they install ROS from debs.